assets/js/base/components/cart-checkout/order-review/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { createSlotFill } from 'wordpress-components';
+import BlockErrorBoundary from '@woocommerce/base-components/block-error-boundary';
+import classnames from 'classnames';
+const { Fill, Slot } = createSlotFill( '__experimentalOrderReview' );
+
+function OrderReview( { children } ) {
+	return <Fill>{ children }</Fill>;
+}
+
+function OrderReviewSlot( { className } ) {
+	return (
+		<Slot
+			bubblesVirtually
+			className={ classnames(
+				className,
+				'wc-block-components-order-reviews'
+			) }
+		>
+			{ ( fills ) => {
+				fills.map( ( fill, i ) => (
+					<BlockErrorBoundary showErrorMessage={ false } key={ i }>
+						{ fill }
+					</BlockErrorBoundary>
```

Comment:
The goal is that this would catch an error but disregard the message and simply not render it.
We can extend this to only show the error if the current user is an admin.